README.rst

```diff
@@ -81,6 +84,18 @@ submodule update --init``. This will fetch the `Makd
 <https://github.com/sociomantic-tsunami/makd>`_ project in ``submodules/makd``.
 
 
+Submodules
+**********
+
+============================= ===========
+Dependency                     Version
+============================= ===========
+makd                           v1.3.x
```

Comment:
Note: it's possible these dependencies are horribly outdated. (Need automation.)

Reply:
Checking out the branch and typing `git submodule` should give a clear picture of what's required.

Reply:
The currently used submodules is different to the minimum required, though, or?

Reply:
@gavin-norman-sociomantic I'm not sure how we can reasonably enforce minimum-required constraints outside of having the submodules at that level.  How is this done?

Reply:
Currently there is no nice way. There's an ongoing discussion about it in an internal repo. I'll send you the link.

Reply:
This is a tough call. As soon as we actually start testing supported version range, we will need that data to be stored in `.neptune.yml` and not in readme. Because of that I am not even sure if we shall describe in standard something that is going to change in next few months.

Reply:
Relevant discussion thread https://github.com/sociomantic/backend/issues/331

Reply:
I guess we can keep it like this until is moved to `.neptune.yml`. Or maybe better start writing it in `.neptune.yml` even if it's not checked yet (although with the risk that we realize get it wrong once doing the implementation and have to change it again anyways).